solana/balance-calculator/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "solana-balance-calculator"
+version = "0.1.0"
```

Comment:
should be `0.0.0` unless this is to published on crates.io
also set `published=false`

Reply:
All other smartmodules have the same issue. If we change here, we should change everywhere, which would move everything backward.

To avoid additional churn, I suggest we leave as is until we transition to packages.